.swiftlint.yml

```diff
@@ -27,9 +30,6 @@ type_body_length:
 file_length:
 - 500
 
-line_length:
-- 120
```

Comment:
Why are we getting rid of this? cc @kevinchan159

Reply:
i spoke w/ @Omarrasheed about it and i think its b/c right now there are hundreds of warnings for this rule if we activated it. I think we should make a task to fix all the warning and add it sometime in the future @Omarrasheed

Reply:
Why not just set it really high, and lower it in the future?

Reply:
we could do that too. @Omarrasheed try to find the smallest number you can use